sapphire/utils.py

```diff
@@ -20,7 +20,7 @@
 ERR = [-1, -999]
 
 #: Speed of light in vacuum in m / ns.
-c = 0.299792458
```

Comment:
Perhaps it should then be renamed to `SPEED_OF_LIGHT`, possibly making the equations less readable.

The annoying things was that some functions define variables `a`, `b`, `c`, `d`, etc. This causes conflicts with the `c`. Perhaps those should use other letter ranges to prevent the conflicts.